### DISCUSSION TITLE
Eso mantiene activo todo el tiempo, como hago para que no se active solo?

### DISCUSSION THREAD

**Discussion by MarieGomezCode** [blueman-project/blueman#2160] [Q&A]
blueman:
BlueZ:
Distribution:
Desktop environment:

* [ ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

<!--

ℹ Please use some service like [Pastebin](http://pastebin.com/) or [GitHub Gist](https://gist.github.com/) to post your logs and keep the thread clean and readable. Make sure to describe the exact steps you took when producing it.

->

**Comment by MarieGomezCode** [comment]
Como hago para que no mantenga activo el bluetooth , quiero que siempre ese desactivado

**Comment by infirit** [comment]
1. https://github.com/blueman-project/blueman/wiki/Troubleshooting#turn-on-bluetooth-device-on-boot (but make sure it's false)
2. Disable automatic powering in the power manager plugin (open plugin window)
